toolkit/components/geometry/src/geometry.udl

```diff
@@ -2,6 +2,8 @@
 namespace geometry {
   double gradient(Line ln);
   Point? intersection(Line ln1, Line ln2);
+
+  string string_round(string s);
```

Comment:
I like this as a short-term solution to testing strings, but ultimately I think we should try to keep the fixtures the same. `coveralls` has should cover the string functionality.  WDYT?